projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html

```diff
@@ -1,6 +1,12 @@
-<nav>
-  <span *ngFor="let crumb of crumbs$ | async">
-    <a [routerLink]="crumb.link" [innerHTML]="crumb.label"></a>
-  </span>
+<nav aria-label="Breadcrumb">
```

Comment:
https://travis-ci.com/github/SAP/spartacus/jobs/519146902#L1575 😉

Reply:
This is works perfect for NVDA and JAWS. Unfortunately, VoiceOver fails to vocalize it, even though it shows up correctly in the rotor. After some research and testing this one below turned out to be a good cross-SR combination:
<img width="783" alt="Screen Shot 2021-06-28 at 1 09 48 PM" src="https://user-images.githubusercontent.com/15113219/123678478-57dd5280-d814-11eb-872e-9244d4efd836.png">